src/expr/src/linear.rs

```diff
@@ -1396,54 +1396,82 @@ pub mod plan {
                     }
                 }
             }
+            // At this point, we are "certain" that `lower_bound_i128` is at
```

Comment:
Nit, but `lower_bound_128` doesn't exist any more.

Reply:
I'll merge as-is because you mentioned you're doing another pass on this soon and can avoid another trip through CI. Glad to fix myself, though, if you'd prefer.